$OpenBSD: patch-deps_eleveldb_c_src_leveldb_Makefile,v 1.1 2013/10/27 11:24:05 jmatthew Exp $

don't run build_detect_platform, it's a waste of time and it
makes ld destroy /dev/null

--- deps/eleveldb/c_src/leveldb/Makefile.orig	Sat Oct 26 22:26:25 2013
+++ deps/eleveldb/c_src/leveldb/Makefile	Sat Oct 26 22:26:13 2013
@@ -15,7 +15,7 @@ OPT ?= -O2 -g -DNDEBUG    # (A) Production use (optimi
 #-----------------------------------------------
 
 # detect what platform we're building on
-$(shell ./build_detect_platform build_config.mk)
+#$(shell ./build_detect_platform build_config.mk)
 # this file is generated by the previous line to set build flags and sources
 include build_config.mk
 
